connect/eaas/core/extension.py

```diff
@@ -80,12 +80,52 @@ def get_variables(cls):
 class EventsApplicationBase(ApplicationBase):
     """Base class to implements an Events Application."""
 
-    def __init__(self, client, logger, config, installation_client=None, installation=None):
+    def __init__(
+        self,
+        client,
+        logger,
+        config,
+        installation_client=None,
+        installation=None,
+        context=None,
+    ):
         self.client = client
         self.logger = logger
         self.config = config
         self.installation_client = installation_client
         self.installation = installation
+        self.context = context
+
+    def get_installation_admin_client(self, installation_id):
+        data = (
```

Comment:
There are 5x repeats of getting such client maybe worth make separate util function like `def get_....(client, context, installation_id, is_async)`?

Reply:
They are slightly different from each other, I will think about that. Thanks)